app/scripts/services/aggregatedLoggingService.js

```diff
@@ -0,0 +1,66 @@
+'use strict';
+
+angular.module("openshiftConsole")
+  .factory("AggregatedLoggingService", function($q, $cacheFactory, Logger, $interval, APIService, DataService) {
+
+    var cachedUserPermissions = $cacheFactory('operationsUsersCache', {
+      number: 10
+    });
+
+    var inFlightPermissionsRequest = null;
+    var authPromise = null;
+
+    // Create SelfSubjectAccessReview request againt authorization API to check whether
+    // current user can view pods/log from 'openshift-logging' project.
+    // Users who can view such logs are 'operations' users
+    var isOperationsUser = function(userName) {
+      var userAllowed = cachedUserPermissions.get(userName);
+
+      if (!userAllowed) {
+        // If a request is already in flight, return the promise for that request.
+        if (inFlightPermissionsRequest) {
+          return inFlightPermissionsRequest;
+        }
+
+        Logger.log("AggregatedLoggingService, loading whether user " + userName + " is Operations user");
+        var ssar = {
+          apiVersion: 'authorization.k8s.io/v1',
+          kind: 'SelfSubjectAccessReview',
+          spec: {
+            resourceAttributes: {
+              resource: 'pods/log',
+              namespace: 'openshift-logging',
```

Comment:
This should be 'default' namespace since this is how we determine similar in the multi-tenant plugin

Reply:
You will also need to set scopes to an empty array like:

* https://github.com/fabric8io/openshift-elasticsearch-plugin/blob/master/src/main/java/io/fabric8/elasticsearch/util/RequestUtils.java#L96
* https://github.com/fabric8io/openshift-elasticsearch-plugin/blob/master/src/main/java/io/fabric8/elasticsearch/plugin/OpenshiftAPIService.java#L116

Reply:
@jcantrill  I didnt find scope in resourceAttributes definition. I assume it's used by the above mentioned java library for something else than the auth api.

https://github.com/kubernetes/kubernetes/blob/master/pkg/apis/authorization/types.go#L80-L98